.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will build a golang project
```

Comment:
The default build system is circleCI. Changing this, should be discussed in an issue first